snapshot/lib/snapshot.rb

```diff
@@ -38,7 +38,8 @@ def snapfile_name
     end
 
     def kill_simulator
-      `killall iOS Simulator &> /dev/null`
+      `killall 'iOS Simulator' &> /dev/null`
+      `killall Simulator &> /dev/null`
```

Comment:
Yes, we found that the original code perhaps only worked by accident...

"iOS Simulator" was the old name of the simulator app, but `killall iOS Simulator` winds up passing two arguments to `killall`, 'iOS' and 'Simulator'. 'iOS' matches nothing, but 'Simulator' matches the current name of the simulator app.

So, to actually kill everything for both known simulator apps, we've changed it to this.